### ISSUE TITLE
How do I add multiple folders and notes to the index?

### ISSUE THREAD

**Issue by jhon65496** [tamlok/vnote#2559]
Subject: How do I add multiple folders and notes to the index?

**Describe the bug**
How do I add several (all) folders and notes to the index with all the subfolders and notes?

**To Reproduce**
As a result of the emergency shutdown of Vnote, all the folders in my Notebook became extinct.
For example, I showed it in the picture.
If I understand correctly, they have deleted from the **index**.

How can I add them all at once to the **index**?

I understand how to add a folder or a note or several folders and notes that are on the same level to the **index**.

But I have more than a thousand folders and notes.
Folders have a certain structure. 
In other words, they have many subfolders.
And you need to open a lot of folders, add each folder and note to the **index** - this is very difficult.

**Expected behavior**
At the discretion of the developer

**Screenshots**
**Picture-1**
![image](https://github.com/user-attachments/assets/c2fec1bd-33b6-4858-b351-210cda663841)

**Picture-2**
![image](https://github.com/user-attachments/assets/3c46a1a3-d1b0-45f0-8cdb-bc7fe5ac811d)

**Desktop (please complete the following information):**
 - OS: Windows 10 x64
 - Version VNote: 3.18.2

**Additional context**
No

**Comment by tamlok** [comment]
Are all the JSON index files gone? That is weird!

If so, maybe you can create a new notebook direclty from the root folder:

![image](https://github.com/user-attachments/assets/c301b1e0-1bd5-420c-aa6b-caf273dcfc66)

Please backup your data before the operation.

**Comment by jhon65496** [comment]
> Are all the JSON index files gone? That is weird!
> 
> If so, maybe you can create a new notebook direclty from the root folder:
> 
> ![image](https://private-user-images.githubusercontent.com/5763157/379759118-c301b1e0-1bd5-420c-aa6b-caf273dcfc66.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjk5MzUyMzAsIm5iZiI6MTcyOTkzNDkzMCwicGF0aCI6Ii81NzYzMTU3LzM3OTc1OTExOC1jMzAxYjFlMC0xYmQ1LTQyMGMtYWE2Yi1jYWYyNzNkY2ZjNjYucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MTAyNiUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDEwMjZUMDkyODUwWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9ZDAyZDI4YmZjMDgwZDQyOGJkMTI1ZGEwNzAwMzY1YjI4YjMwOTllMmQzODkzMTk0MjhhNGI3ZDA4OGFkOTQxZiZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.leTYn0u_-Hz41QrMUNtMWuy518XrzYc32JzqzsEczeE)
> 
> Please backup your data before the operation.

The issue is closed.
Your decision helped. 
Everything works.

Thank you.
